services/dynamic-scheduler/src/simcore_service_dynamic_scheduler/services/director_v0/_thin_client.py

```diff
@@ -0,0 +1,47 @@
+import urllib.parse
+
+from common_library.unset import as_dict_exclude_none
+from fastapi import status
+from httpx import Response
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services_base import ServiceKeyVersion
+from models_library.users import UserID
+from servicelib.fastapi.app_state import SingletonInAppStateMixin
+from servicelib.fastapi.http_client import AttachLifespanMixin
+from servicelib.fastapi.http_client_thin import (
+    BaseThinClient,
+    expect_status,
+    retry_on_errors,
+)
+from yarl import URL
+
+
+class DirectorV0ThinClient(
+    SingletonInAppStateMixin, BaseThinClient, AttachLifespanMixin
+):
+    app_state_name: str = "director_v0_thin_client"
+
+    @retry_on_errors()
+    @expect_status(status.HTTP_200_OK)
+    async def get_running_interactive_service_details(
+        self, node_id: NodeID
+    ) -> Response:
+        return await self.client.get(f"/running_interactive_services/{node_id}")
+
+    @retry_on_errors()
+    @expect_status(status.HTTP_200_OK)
+    async def get_services_labels(self, service: ServiceKeyVersion) -> Response:
```

Comment:
why do you need this? this is exposed by the catalog right?

Reply:
Q: why were we using it via director-v0 then inside here?

Reply:
labels was now moved to the catalog and will be used form there. removing from this client